packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -10,27 +23,32 @@
     cursor-pointer
     transition-default
     focus-base
-    no-underline
-    text-0h;
+    no-underline;
   color: inherit;
-  border-block-end: 2px solid transparent;
+  font-size: var(--calcite-font-size-0);
+  line-height: var(--calcite-font-line-height-fixed-lg);
+  border-block-end: var(--calcite-spacing-base) solid
+    var(--calcite-navigation-logo-border-color, var(--calcite-internal-navigation-logo-border-color, transparent));
+  background-color: var(--calcite-navigation-logo-background-color, var(--calcite-color-foreground-1));
 }
 
 .anchor:hover,
 .anchor:focus {
-  @apply bg-foreground-2;
+  background-color: var(--calcite-navigation-logo-background-color-hover, var(--calcite-color-foreground-2));
```

Comment:
Should this be `--calcite-navigation-logo-background-color-focus` ? keeping https://github.com/Esri/calcite-design-system/issues/10002 removes background-color on :hover

Reply:
same as above. I don't think state tokens are necessary here but if they are, first check if the navigation-logo background color is different than navigation? If not, use the same token for both --calcite-navigation-background-color-hover